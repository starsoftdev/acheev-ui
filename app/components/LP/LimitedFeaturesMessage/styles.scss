@import '~styles/settings/settings';

.limitedFeaturesMessage {
  padding-top: $rem-md;
  background-color: $secondary;

  &__title {
    font-size: $fs-lg;
    color: $lightest-gray;
  }

  &__message,
  &__link {
    color: $light-gray;
  }

  &__link {
    text-transform: none;
    text-decoration: none;
    font-size: $fs-base;
    font-weight: normal;
  }
}
