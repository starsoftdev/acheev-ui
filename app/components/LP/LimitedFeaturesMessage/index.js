// @flow
import React from 'react';
import './styles.scss';

type Props = {
  role?: string,
};

const LimitedFeaturesMessage = ({ role }: Props) =>
  role === 'business' && (
    <section className="limitedFeaturesMessage">
      <div className="row">
        <div className="small-12 column">
          <h1 className="limitedFeaturesMessage__title">
            Sorry, you&apos;re not a Lift Rewards Pro Partner so we can only
            show you an example template of this section!
          </h1>
          <p className="limitedFeaturesMessage__message">
            Lift Rewards Pro Partners have access to our full analytics
            dashboard, which offers patient and product insights aggregated from
            the largest database of consumer-reported information for legal
            cannabis products. For more information or to sign up, contact us at{' '}
            <a
              className="limitedFeaturesMessage__link"
              href="mailto:hello@lift.co"
            >
              hello@lift.co
            </a>
          </p>
        </div>
      </div>
    </section>
  );

export default LimitedFeaturesMessage;
