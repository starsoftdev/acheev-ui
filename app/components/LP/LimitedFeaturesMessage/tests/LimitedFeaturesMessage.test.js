import React from 'react';
import { create } from 'react-test-renderer';

import LimitedFeaturesMessage from 'components/LP/LimitedFeaturesMessage';

describe('<LimitedFeaturesMessage />', () => {
  it('Should match snapshoot for business', () => {
    const component = create(<LimitedFeaturesMessage role="business" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should match snapshoot for pro', () => {
    const component = create(<LimitedFeaturesMessage role="pro" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
