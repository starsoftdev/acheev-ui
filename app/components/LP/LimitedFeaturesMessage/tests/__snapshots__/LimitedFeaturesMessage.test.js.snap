// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LimitedFeaturesMessage /> Should match snapshoot for business 1`] = `
<section
  className="limitedFeaturesMessage"
>
  <div
    className="row"
  >
    <div
      className="small-12 column"
    >
      <h1
        className="limitedFeaturesMessage__title"
      >
        Sorry, you're not a Lift Rewards Pro Partner so we can only show you an example template of this section!
      </h1>
      <p
        className="limitedFeaturesMessage__message"
      >
        Lift Rewards Pro Partners have access to our full analytics dashboard, which offers patient and product insights aggregated from the largest database of consumer-reported information for legal cannabis products. For more information or to sign up, contact us at
         
        <a
          className="limitedFeaturesMessage__link"
          href="mailto:hello@lift.co"
        >
          hello@lift.co
        </a>
      </p>
    </div>
  </div>
</section>
`;

exports[`<LimitedFeaturesMessage /> Should match snapshoot for pro 1`] = `null`;
